components/Chips/src/MDCChipView.h

```diff
@@ -110,6 +110,14 @@
  */
 @property(nonatomic, strong, nullable) UIFont *titleFont UI_APPEARANCE_SELECTOR;
 
+/*
+ This property determines if an @c MDCChipView should use the @c MDCInkView behavior or not.
+ By setting this property to @c YES, @c MDCStatefulRippleView is used to provide the user visual
+ touch feedback, instead of the legacy @c MDCInkView.
+ @note Defaults to @c NO.
+ */
+@property(nonatomic, assign) BOOL enableRippleBehavior;
```

Comment:
When adding custom states - I recommend adding getters/setters for them in the API so they can be tested or used in examples.

Reply:
Adding new API for new states is out of scope for this PR.